source/base/Text.ooc

```diff
@@ -134,6 +134,13 @@ Text: cover {
 	free: func@ ~withCriteria (criteria: Owner) -> Bool {
 		this _buffer free(criteria)
 	}
+	slice: func ~untilEnd (start: Int) -> This {
+		result := This new(this _buffer slice(start))
+		if (this _buffer owner == Owner Receiver)
+			result = result copy() // TODO: Could we be smarter here?
```

Comment:
Also, in C# we had an implementation that worked like in PHP, which was pretty neat:
http://php.net/manual/en/function.substr.php

Obviously, don't implement it in this PR.